magicgui/tqdm.py

```diff
@@ -0,0 +1,151 @@
+"""A wrapper around the tqdm.tqdm iterator that adds a ProgressBar to a magicgui."""
```

Comment:
Just wondering if there is another place this could live that is not at the top level, i.e. a grouping of a larger thing that it might be an example of? I guess though there isn't anything else quite like in magicgui right now as it's not a widget per se (that is `ProgressBar` itself, this is the iterator) but something that if found inside magic functions will generate a widget?

Reply:
ha! triplicate of above, my github was clearly acting up!!!